contentcuration/contentcuration/static/js/edit_channel/new_channel/hbtemplates/channel_container.handlebars

```diff
@@ -12,15 +12,13 @@
 	{{/if}}
 </div>
 {{#if edit}}
+	<p id="channel_name_error">Channel must have a name</p>
```

Comment:
Added to support model changes for https://trello.com/c/U3bvp4rP/227-editors-should-use-placeholder-text-that-disappears-when-the-user-starts-typing